test/test_encryption.py

```diff
@@ -2494,5 +2494,195 @@ def MongoClient(**kwargs):
         client_encryption.close()
 
 
+# https://github.com/mongodb/specifications/blob/d4c9432/source/client-side-encryption/tests/README.rst#explicit-encryption
```

Comment:
I think this link needs a more recent sha, I don't see these tests in that commit.

Reply:
Done.